pnpm-workspace.yaml

```diff
@@ -1,2 +1,8 @@
 packages:
-  - './packages/*'
\ No newline at end of file
+  - './packages/*'
+
+autoInstallPeers: false
+linkWorkspacePackages: deep
```

Comment:
I believe the default behavior in pnpm 7 might have merely been `true`, but regardless, `deep` is very desirable since packages in this repo have dependencies on their siblings that go more than one layer deep.